.github/workflows/build_and_run_test.yml

```diff
@@ -15,20 +15,16 @@ jobs:
     runs-on: self-hosted
 
     steps:
-    - uses: actions/checkout@v2
+    - name: Checkout codes
+      uses: actions/checkout@v2
       with:
         submodules: true
-    - name: Add wasm toolchain
-      uses: actions-rs/toolchain@v1
-      with:
-        toolchain: stable
-        target: wasm32-unknown-unknown
-        override: true
-    - name: Build relay node and parachain
+
+    - name: Build docker image
       run: |
-        cargo build --release
-        cd polkadot
-        cargo build --release
+        docker images
```

Comment:
I don't know why you need to run `docker images` before `build.sh`. For debugging ?

Reply:
> I don't know why you need to run `docker images` before `build.sh`. For debugging ?

mostly yes, `build.sh` will pull base images and build images that might result in dangling images, I wanted to know stats before it

Reply:
I removed it in latest commit